README.md

```diff
@@ -35,6 +35,7 @@ export interface CounterMutations {
   inc: {
     amount: number
   }
+  reset: void // having no payload
```

Comment:
```suggestion
  reset: undefined // having no payload
```

`void` is not `null` nor `undefined`. It means that the value *will not be used* that only makes sense as a function return type in most cases.

Reply:
According to the TypeScript documentation, I understood void to mean "the absence of having any type at all".

https://www.typescriptlang.org/docs/handbook/basic-types.html#void

> You may commonly see this as a return type of function that do not return a value:

It can also be interpreted as not necessarily being limited to the return value of a function, however we may commonly see this as the return type of functions that do not return a value.

Therefore, I thought it would be more explicit to define "absence" with `void` rather than using `undefined` to indicate "not defined".

---

However, I am aware that I am not familiar with the subject and I am not obsessed with void.

So, if you still think "undefined" is more appropriate after seeing my comments, I'll stand by it.

Reply:
I see. That makes sense to me.